airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1182,6 +1184,52 @@ paths:
           $ref: "#/components/responses/NotFoundResponse"
         "422":
           $ref: "#/components/responses/InvalidInputResponse"
+  /v1/source_oauths/get_oauth_consent:
+    post:
```

Comment:
it feels like this should be a GET so the server redirects the user directly, but it would break our API convention. With this being a post, the UI would need to redirect the user. NBD I think.